### PR TITLE
refactor: add A01

### PR DESCRIPTION
**Pull request by Lash-L**

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -1,4 +1,21 @@
+import {
```
Comment (humbertogontijo) [inline commitlint.config.js]: A check is failing because of this import. Everything else looks good

Comment (Lash-L) [reply commitlint.config.js]: Looks like it should be passing now.

**Review by humbertogontijo — DISMISSED** [review]
Looking good

**Comment by Lash-L** [comment]
Okay I finally figured out how to get the major release to work, so I just need another approval and I can fix it. You can ignore commitlint as I will squash commits anyways.

**Review by humbertogontijo — APPROVED** [review]

**Comment by Lash-L** [comment]
coudl be both things
image

**Comment by Lash-L** [comment]
I removed the old release and tag but I think for semantic release to do it all you need to remove the last 1.0.0 commit. I'm unable to do that

**Comment by humbertogontijo** [comment]
I deleted the last commit and its running again

**Comment by Lash-L** [comment]
> I deleted the last commit and its running again

looks like it worked - thanks

**Pull request merged**